test.sh

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env bash
 
+set -euo pipefail
+
 ROOT=$(cd "$(dirname ${BASH_SOURCE[0]:-$0})"; pwd)
 
 cd "$ROOT"
-go test $(find . -name '*_test.go' | sed 's/[^/]\+$//' | uniq)
+
+go get -d .
+
+for path in $(find . -name '*_test.go' | sed 's/[^/]\+$//' | uniq); do
+    go test "$path"
```

Comment:
You should be able to do something like `go test ./...` from the root directory to run all tests. Here's a quick reference: https://stackoverflow.com/questions/42691705/what-does-mean-in-go

Reply:
Oh, I've always thought that `...` in go is just "everything in that directory", I've learned something new!